### ISSUE TITLE
Tooltip not read by screen reader on Windows Desktop

### ISSUE THREAD

**Issue by bourdakos1** [flutter/flutter#112876]
## Steps to Reproduce

1. Turn on VoiceOver/NVDA/JAWS
2. Execute `flutter run` on the code sample
3. Hover over the icon button
4. Click the icon button

**Expected results:**

3. The button should be read as "this is a tooltip button" when hovered

**Actual results:**

3. The button is read as "this is a tooltip button" on macOS, but read as "button" on Windows

<details>
<summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';
import 'package:flutter/semantics.dart';

void main() => runApp(const MyApp());

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Code Sample',
      home: Scaffold(
        body: IconButton(
          icon: const Icon(Icons.settings),
          tooltip: 'This is a tooltip',
          onPressed: () {
            SemanticsService.announce(
              'This is an announcement',
              TextDirection.ltr,
            );
          },
        ),
      ),
    );
  }
}
```

</details>

<details>
  <summary>Logs</summary>

```
[  +10 ms] executing: [C:\Users\bourdakos\Documents\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +205 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms] 128b94e247326ecdcbe954febb73adbac4b2449c
[        ] executing: [C:\Users\bourdakos\Documents\flutter/] git tag --points-at 128b94e247326ecdcbe954febb73adbac4b2449c
[ +215 ms] Exit code 0 from: git tag --points-at 128b94e247326ecdcbe954febb73adbac4b2449c
[   +2 ms] executing: [C:\Users\bourdakos\Documents\flutter/] git describe --match *.*.* --long --tags
128b94e247326ecdcbe954febb73adbac4b2449c
[ +247 ms] Exit code 0 from: git describe --match *.*.* --long --tags 128b94e247326ecdcbe954febb73adbac4b2449c
[        ] 3.4.0-39.0.pre-7-g128b94e247
[  +19 ms] executing: [C:\Users\bourdakos\Documents\flutter/] git rev-parse --abbrev-ref --symbolic @{upstream}
[ +184 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{upstream}
[        ] origin/master
[        ] executing: [C:\Users\bourdakos\Documents\flutter/] git ls-remote --get-url origin
[ +213 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[ +207 ms] executing: [C:\Users\bourdakos\Documents\flutter/] git rev-parse --abbrev-ref HEAD
[ +180 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] master
[ +125 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +397 ms] executing: C:\Users\bourdakos\platform-tools_r31.0.3-windows\platform-tools\adb.exe devices -l
[ +151 ms] List of devices attached
[   +5 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +95 ms] Multiple devices found:
[ +101 ms] Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19042.2006]
[   +1 ms] Chrome (web)      • chrome  • web-javascript • Google Chrome 106.0.5249.91
[        ] Edge (web)        • edge    • web-javascript • Microsoft Edge 102.0.1245.41
[        ] [1]: Windows (windows)
[        ] [2]: Chrome (chrome)
[        ] [3]: Edge (edge)
[   +3 ms] Please choose one (To quit, press "q/Q")
[        ] : 
[+4988 ms] 1
[  +19 ms] Skipping pub get: version match.
[ +162 ms] Generating
C:\Users\bourdakos\Documents\bug_repro\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +143 ms] Initializing file store
[  +16 ms] Skipping target: gen_localizations
[  +16 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents:C:\Users\bourdakos\Documents\bug_repro\.dart_tool\package_config_subset}
[  +41 ms] gen_dart_plugin_registrant: Complete
[   +5 ms] Skipping target: _composite
[   +4 ms] complete
[  +11 ms] Launching lib\main.dart on Windows in debug mode...
[   +8 ms] C:\Users\bourdakos\Documents\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev
C:\Users\bourdakos\Documents\flutter\bin\cache\dart-sdk\bin\snapshots\frontend_server.dart.snapshot --sdk-root
C:\Users\bourdakos\Documents\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter
--experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill
C:\Users\BOURDA~1\AppData\Local\Temp\flutter_tools.9b9a3dc1\flutter_tool.9a0fc0ca\app.dill --packages
C:\Users\bourdakos\Documents\bug_repro\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts 
--track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill
build\c075001b96339384a97db4862b8ab8db.cache.dill.track.dill --verbosity=error --flutter-widget-cache
--enable-experiment=alternative-invalidation-strategy
[  +46 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest
-version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[ +152 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8
-latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[   +1 ms] [
             {
               "instanceId": "4228c84f",
               "installDate": "2022-01-21T18:11:43Z",
               "installationName": "VisualStudio/16.11.9+32106.194",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Professional",
               "installationVersion": "16.11.32106.194",
               "productId": "Microsoft.VisualStudio.Product.Professional",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Professional\\Common7\\IDE\\devenv.exe",        
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Professional 2019",
               "description": "Professional IDE best suited to small teams",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual
               Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-v16.11#16.11.9",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2022-01-21T18:11:43.4743112Z",
               "catalog": {
                 "buildBranch": "d16.11",
                 "buildVersion": "16.11.32106.194",
                 "id": "VisualStudio/16.11.9+32106.194",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.11.9",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "9",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.11.9+32106.194",
                 "requiredEngineVersion": "2.11.52.58712"
               },
               "properties": {
                 "campaignId": "",
                 "channelManifestId": "VisualStudio.16.Release/16.11.9+32106.194",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +8 ms] Building Windows application...
[ +433 ms] <- compile package:bug_repro/main.dart
[   +2 ms] executing: C:\Program Files (x86)\Microsoft Visual
Studio\2019\Professional\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S
C:\Users\bourdakos\Documents\bug_repro\windows -B build\windows -G Visual Studio 16 2019
[ +243 ms] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19042.
[  +57 ms] -- Configuring done
[ +309 ms] -- Generating done
[  +66 ms] -- Build files have been written to: C:/Users/bourdakos/Documents/bug_repro/build/windows
[  +10 ms] executing: C:\Program Files (x86)\Microsoft Visual
Studio\2019\Professional\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug --targetINSTALL --verbose
[ +657 ms] Microsoft (R) Build Engine version 16.11.2+f32259642 for .NET Framework
[   +1 ms] Copyright (C) Microsoft Corporation. All rights reserved.
[ +381 ms] Build started 10/4/2022 4:04:03 PM.
[ +874 ms] Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[   +1 ms] Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\INSTALL.vcxproj" (1) is building
"C:\Users\bourdakos\Documents\bug_repro\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +57 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[ +308 ms]   Checking Build System
[ +189 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +5 ms] Done Building Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +127 ms] Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\INSTALL.vcxproj" (1) is building
"C:\Users\bourdakos\Documents\bug_repro\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[   +1 ms] Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\bourdakos\Documents\bug_repro\build\windows\runner\bug_repro.vcxproj" (4) on node 1 (default targets).
[   +1 ms] Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\runner\bug_repro.vcxproj" (4) is building
"C:\Users\bourdakos\Documents\bug_repro\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.       
[   +7 ms] CustomBuild:
[        ]   "The build of
'C:\Users\bourdakos\Documents\bug_repro\build\windows\CMakeFiles\9c4fc6f798b73840992bb3ce2f7afe41\flutter_assemble.rule' depends on    
'C:\USERS\BOURDAKOS\DOCUMENTS\BUG_REPRO\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'C:\Users\bourdakos\Documents\bug_repro\build\windows\CMakeFiles\c21b719c1bb1867fca19a7ea97cd8309\flutter_windows.dll.rule'. The items 
cannot be built in parallel."
[ +190 ms]   Generating C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/flutter_windows.dll,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/flutter_export.h,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/flutter_windows.h,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/flutter_messenger.h,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/flutter_plugin_registrar.h,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/flutter_texture_registrar.h,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc,
C:/Users/bourdakos/Documents/bug_repro/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+1702 ms]   [   +6 ms] executing: [C:\Users\bourdakos\Documents\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +215 ms]   [ +235 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms]   [   +1 ms] 128b94e247326ecdcbe954febb73adbac4b2449c
[        ]   [        ] executing: [C:\Users\bourdakos\Documents\flutter/] git tag --points-at 128b94e247326ecdcbe954febb73adbac4b2449c[ +221 ms]   [ +222 ms] Exit code 0 from: git tag --points-at 128b94e247326ecdcbe954febb73adbac4b2449c
[   +1 ms]   [   +1 ms] executing: [C:\Users\bourdakos\Documents\flutter/] git describe --match *.*.* --long --tags
128b94e247326ecdcbe954febb73adbac4b2449c
[ +233 ms]   [ +232 ms] Exit code 0 from: git describe --match *.*.* --long --tags 128b94e247326ecdcbe954febb73adbac4b2449c
[        ]   [        ] 3.4.0-39.0.pre-7-g128b94e247
[  +17 ms]   [  +17 ms] executing: [C:\Users\bourdakos\Documents\flutter/] git rev-parse --abbrev-ref HEAD
[ +180 ms]   [ +180 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms]   [        ] master
[ +130 ms]   [ +132 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms]   [   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +122 ms]   [ +123 ms] executing: [C:\Users\bourdakos\Documents\flutter/] git rev-parse --abbrev-ref --symbolic @{upstream}
[ +187 ms]   [ +187 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{upstream}
[        ]   [        ] origin/master
[        ]   [        ] executing: [C:\Users\bourdakos\Documents\flutter/] git ls-remote --get-url origin
[ +194 ms]   [ +195 ms] Exit code 0 from: git ls-remote --get-url origin
[        ]   [        ] https://github.com/flutter/flutter.git
[        ]   [        ] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +1 ms]   [   +3 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[ +463 ms]   [ +468 ms] Initializing file store
[  +18 ms]   [  +17 ms] Done initializing file store
[  +61 ms]   [  +61 ms] Skipping target: gen_localizations
[  +17 ms]   [  +17 ms] Skipping target: gen_dart_plugin_registrant
[+4000 ms]   [+4001 ms] Skipping target: unpack_windows
[ +640 ms]   [ +640 ms] Skipping target: kernel_snapshot
[ +310 ms]   [ +310 ms] Skipping target: debug_bundle_windows_assets
[   +1 ms]   [        ] Persisting file store
[  +20 ms]   [  +20 ms] Done persisting file store
[  +10 ms]   [  +10 ms] build succeeded.
[  +17 ms]   [  +17 ms] "flutter assemble" took 6,382ms.
[ +225 ms]   [ +225 ms] ensureAnalyticsSent: 217ms
[   +1 ms]   [        ] Running 0 shutdown hooks
[        ]   [        ] Shutdown hooks complete
[        ]   [        ] exiting with code 0
[+1159 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\flutter\flutter_assemble.vcxproj" (default      
targets).
[  +64 ms] Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\runner\bug_repro.vcxproj" (4) is building
"C:\Users\bourdakos\Documents\bug_repro\build\windows\flutter\flutter_wrapper_app.vcxproj" (6) on node 1 (default targets).
[   +2 ms] InitializeBuildStatus:
[   +1 ms]   Creating "flutter_wrapper_app.dir\Debug\flutter_.745470B3.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.   
[   +3 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +65 ms] ClCompile:
[   +1 ms]   All outputs are up-to-date.
[  +51 ms] Lib:
[   +2 ms]   All outputs are up-to-date.
[   +1 ms]   flutter_wrapper_app.vcxproj -> C:\Users\bourdakos\Documents\bug_repro\build\windows\flutter\Debug\flutter_wrapper_app.lib 
[  +22 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.745470B3.tlog\unsuccessfulbuild".
[   +2 ms]   Touching "flutter_wrapper_app.dir\Debug\flutter_.745470B3.tlog\flutter_wrapper_app.lastbuildstate".
[   +3 ms] Done Building Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\flutter\flutter_wrapper_app.vcxproj" (default   
targets).
[  +16 ms] InitializeBuildStatus:
[   +1 ms]   Creating "bug_repro.dir\Debug\bug_repro.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +6 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +47 ms] ClCompile:
[   +1 ms]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\CL.exe /c
/IC:\Users\bourdakos\Documents\bug_repro\windows /IC:\Users\bourdakos\Documents\bug_repro\windows\flutter\ephemeral
/IC:\Users\bourdakos\Documents\bug_repro\windows\flutter\ephemeral\cpp_client_wrapper\include /Zi /nologo /W4 /WX /diagnostics:column  
/Od /Ob0 /D WIN32 /D _WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D "FLUTTER_VERSION=\"1.0.0+1\"" /D FLUTTER_VERSION_MAJOR=1 /D
FLUTTER_VERSION_MINOR=0 /D FLUTTER_VERSION_PATCH=0 /D FLUTTER_VERSION_BUILD=1 /D NOMINMAX /D UNICODE /D _UNICODE /D
"CMAKE_INTDIR=\"Debug\"" /D _UNICODE /D UNICODE /Gm- /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR
/std:c++17 /Fo"bug_repro.dir\Debug\\" /Fd"bug_repro.dir\Debug\vc142.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue
C:\Users\bourdakos\Documents\bug_repro\windows\flutter\generated_plugin_registrant.cc
C:\Users\bourdakos\Documents\bug_repro\windows\runner\flutter_window.cpp
[ +319 ms]   generated_plugin_registrant.cc
[ +289 ms]   flutter_window.cpp
[ +875 ms]   Generating Code...
[ +197 ms] ResourceCompile:
[   +1 ms]   All outputs are up-to-date.
[  +51 ms] Link:
[        ]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\link.exe
/ERRORREPORT:QUEUE /OUT:"C:\Users\bourdakos\Documents\bug_repro\build\windows\runner\Debug\bug_repro.exe" /INCREMENTAL
/ILK:"bug_repro.dir\Debug\bug_repro.ilk" /NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib dwmapi.lib
C:\Users\bourdakos\Documents\bug_repro\windows\flutter\ephemeral\flutter_windows.dll.lib kernel32.lib user32.lib gdi32.lib winspool.libshell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'"      
/manifest:embed /manifestinput:C:\Users\bourdakos\Documents\bug_repro\windows\runner\runner.exe.manifest /DEBUG
/PDB:"C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/bug_repro.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE        
/NXCOMPAT /IMPLIB:"C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/bug_repro.lib" /MACHINE:X64  /machine:x64
bug_repro.dir\Debug\Runner.res
[   +2 ms]   bug_repro.dir\Debug\flutter_window.obj
[   +1 ms]   bug_repro.dir\Debug\main.obj
[   +2 ms]   bug_repro.dir\Debug\utils.obj
[   +1 ms]   bug_repro.dir\Debug\win32_window.obj
[        ]   bug_repro.dir\Debug\generated_plugin_registrant.obj
[ +745 ms]   bug_repro.vcxproj -> C:\Users\bourdakos\Documents\bug_repro\build\windows\runner\Debug\bug_repro.exe
[  +12 ms] FinalizeBuildStatus:
[        ]   Deleting file "bug_repro.dir\Debug\bug_repro.tlog\unsuccessfulbuild".
[   +3 ms]   Touching "bug_repro.dir\Debug\bug_repro.tlog\bug_repro.lastbuildstate".
[   +3 ms] Done Building Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\runner\bug_repro.vcxproj" (default targets).
[  +78 ms] Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\bourdakos\Documents\bug_repro\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (7) on node 1 (default targets).
[   +4 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.4306E078.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.[   +8 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +16 ms] ClCompile:
[   +1 ms]   All outputs are up-to-date.
[   +7 ms] Lib:
[   +1 ms]   All outputs are up-to-date.
[        ]   flutter_wrapper_plugin.vcxproj ->
C:\Users\bourdakos\Documents\bug_repro\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[  +12 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.4306E078.tlog\unsuccessfulbuild".
[   +2 ms]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.4306E078.tlog\flutter_wrapper_plugin.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (defaulttargets).
[  +17 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +12 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[ +319 ms]   Building Custom Rule C:/Users/bourdakos/Documents/bug_repro/windows/CMakeLists.txt
[ +231 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[   +2 ms] Done Building Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\ALL_BUILD.vcxproj" (default targets).
[  +13 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[ +369 ms] PostBuildEvent:
[   +1 ms]   setlocal
[        ]   "C:\Program Files (x86)\Microsoft Visual
Studio\2019\Professional\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P cmake_install.cmake   
[   +1 ms]   if %errorlevel% neq 0 goto :cmEnd
[   +1 ms]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[   +1 ms]   :cmErrorLevel
[   +2 ms]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[ +458 ms]   -- Install configuration: "Debug"
[  +45 ms]   -- Up-to-date: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/icudtl.dat
[   +1 ms]   -- Up-to-date: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/flutter_windows.dll
[  +68 ms]   -- Installing: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets
[  +16 ms]   -- Installing: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[   +7 ms]   -- Installing: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[  +11 ms]   -- Installing: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/fonts
[  +25 ms]   -- Installing:
C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[  +43 ms]   -- Installing: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[ +566 ms]   -- Installing: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/NOTICES.Z
[   +8 ms]   -- Installing: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/packages
[  +17 ms]   -- Installing:
C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[  +30 ms]   -- Installing:
C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[  +26 ms]   -- Installing:
C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[  +19 ms]   -- Installing: C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/shaders
[  +27 ms]   -- Installing:
C:/Users/bourdakos/Documents/bug_repro/build/windows/runner/Debug/data/flutter_assets/shaders/ink_sparkle.frag
[  +73 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[   +2 ms] Done Building Project "C:\Users\bourdakos\Documents\bug_repro\build\windows\INSTALL.vcxproj" (default targets).
[  +41 ms] Build succeeded.
[   +1 ms]     0 Warning(s)
[   +2 ms]     0 Error(s)
[   +1 ms] Time Elapsed 00:00:17.39
[ +106 ms] Building Windows application... (completed in 19.6s)
[+2039 ms] [ERROR:flutter/shell/platform/windows/direct_manipulation.cc(137)] CoCreateInstance(CLSID_DirectManipulationManager,
nullptr, CLSCTX_INPROC_SERVER, IID_IDirectManipulationManager, &manager_) failed
[ +229 ms] Observatory URL on device: http://127.0.0.1:59602/jS2LyrBgPNo=/
[  +10 ms] Caching compiled dill
[ +160 ms] Connecting to service protocol: http://127.0.0.1:59602/jS2LyrBgPNo=/
[ +314 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at
http://127.0.0.1:59602/jS2LyrBgPNo=/.
[ +171 ms] DDS is listening at http://127.0.0.1:59608/grlPJj4L7xQ=/.
[ +131 ms] Successfully connected to service protocol: http://127.0.0.1:59602/jS2LyrBgPNo=/
[+1083 ms] DevFS: Creating new filesystem on the device (null)
[  +48 ms] DevFS: Created new filesystem on the device (file:///C:/Users/BOURDA~1/AppData/Local/Temp/bug_repro88012d5e/bug_repro/)
[  +14 ms] Updating assets
[ +211 ms] Syncing files to device Windows...
[   +7 ms] Compiling dart to kernel with 0 updated files
[   +1 ms] Processing bundle.
[   +5 ms] <- recompile package:bug_repro/main.dart e3905239-7864-447b-9c52-275ddd9b02be
[   +2 ms] <- e3905239-7864-447b-9c52-275ddd9b02be
[  +14 ms] Bundle processing done.
[ +189 ms] Updating files.
[   +1 ms] DevFS: Sync finished
[   +5 ms] Syncing files to device Windows... (completed in 222ms)
[   +4 ms] Synced 0.0MB.
[   +6 ms] <- accept
[   +4 ms] Connected to _flutterView/0x262611bf5c0.
[   +9 ms] Flutter run key commands.
[   +5 ms] r Hot reload. 
[   +2 ms] R Hot restart.
[   +1 ms] h List all available interactive commands.
[   +1 ms] d Detach (terminate "flutter run" but leave application running).
[   +1 ms] c Clear the screen
[   +1 ms] q Quit (terminate the application on the device).
[   +1 ms]  Running with sound null safety 
[   +3 ms] An Observatory debugger and profiler on Windows is available at: http://127.0.0.1:59608/grlPJj4L7xQ=/
[ +116 ms] The Flutter DevTools debugger and profiler on Windows is available at:
http://127.0.0.1:9101?uri=http://127.0.0.1:59608/grlPJj4L7xQ=/
```

```
Analyzing bug_repro...                                                  
No issues found! (ran in 3.3s)
```

```
[√] Flutter (Channel master, 3.4.0-40.0.pre.7, on Microsoft Windows [Version 10.0.19042.2006], locale en-US)
    • Flutter version 3.4.0-40.0.pre.7 on channel master at C:\Users\bourdakos\Documents\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 128b94e247 (43 minutes ago), 2022-10-04 15:06:53 -0400
    • Engine revision 256b8d066f
    • Dart version 2.19.0 (build 2.19.0-272.0.dev)
    • DevTools version 2.18.0

[√] Windows Version (Installed version of Windows is version 10 or higher)

[X] Android toolchain - develop for Android devices
    • Android SDK at C:\Users\bourdakos\platform-tools_r31.0.3-windows
    X cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Professional 2019 16.11.9)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional
    • Visual Studio Professional 2019 version 16.11.32106.194
    • Windows 10 SDK version 10.0.19041.0

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] VS Code (version 1.71.2)
    • VS Code at C:\Users\bourdakos\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.48.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19042.2006]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 106.0.5249.91
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 102.0.1245.41

[√] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 2 categories.
```

</details>

**Comment by yaakovschectman** [comment]
It appears we are currently not using the tooltip in on the engine; there is no `tooltip` member of `SemanticsNode`. Adding this looks straightforward, but we need some way of presenting it in the resulting MSAA node. What property do we actually want to populate with the tooltip value? Incorporate it into either the beginning or end of the Name of the MSAA node?

**Comment by yaakovschectman** [comment]
I will separate the announcement side of this issue to a separate issue.

**Comment by yaakovschectman** [comment]
Tooltips have been added to the narration.
